### DISCUSSION TITLE
نمایش تاریخ روز به صورت پیش فرض

### DISCUSSION THREAD

**Discussion by HMP1989** [majidh1/JalaliDatePicker#67] [Question]
سلام وقتتون بخیر. خسته نباشین. چطوری میتونم به محض اینکه صفحه لود میشه، و قبل از اینکه کاربر روی اینپوت بزنه و تقویم براش باز بشه، تاریخ روز جاری، در آن به صورت خودکار به نمایش دربیاد. به طور کلی، تاریخ روز به صورت پیشفرض نمایش داده بشه. مقدور هست یه مثال هم بزنید

**Comment by majidh1** [comment]
سلام وقت بخیر
میتونید از متد show استفاده کنید و اینپوت مد نظرتون رو به این متد ارسال کنید. 
```js
jalaliDatepicker.show(document.getElementById("input_auto_show"));
```

برای اینکه بعد از لود صفحه دیتپیکر باز بشه میتونید از ایونت لود داکیومنت استفاده کنید. مثال:

```js
document.addEventListener("DOMContentLoaded", function(){
    jalaliDatepicker.show(document.getElementById("input_auto_show")); //select input by Id `input_auto_show`
});
```

**Reply by HMP1989** [reply]
از پاسخگویی شما بسیار متشکرم.
آیا راهی وجود داره بتونم برای اینپوتم، هم برای زمان و هم برای تاریخ، برای داخل خود اینپوت ها، initialValue تعریف کنم. قبل از اینکه کاربر بخواهد هر تعاملی با سایت باشه، یه initialValue به صورت اتومات برای تاریخ جاری، و initialValue به صورت اتومات برای زمان جاری، به نمایش درآمده باشه. همونطور که مثلا وقتی تقویم رو باز می کنیم، یه فوکوسی روی تاریخ جاری هست، همون تاریخ جاری، به صورت پیشفرض و اتومات، داخل اینپوت به نمایش درآمده باشه و چاپ شده باشه.  یا مثلا وقتی روی زمان کلیک می کنیم و زمان رو به صورت کاملا دقیق داره نشون میده، همون زمان به صورت پیش فرض و اتومات، داخل اینپوت مربوطه اش، به نمایش درآمده باشه و چاپ شده باشه

**Reply by majidh1** [reply]
مقدار دهی پیشفرض توسط دیتپیکر انجام نمیشه ولی شما قبل از نمایش دیتپیکر میتونید اینپوت را مقدار دهی کنید در ایونت DOMContentLoaded قبل از متد show اینکار را انجام دهید

```js
document.getElementById("input_auto_show").value = new Date().toLocaleString('fa-IR-u-nu-latn').replace(",","")
```

**Reply by Absalamyar** [reply]
درست شد تشکر بسیار زیاد از رهنمایی شما دوست عزیز
